diff --git a/dist/sql-wasm.js b/dist/sql-wasm.js
index 6ef6c1f54a368431b22dea4da123f1341b4a1780..8dd8a53cefa7779f0556eb1038be35ad547beabb 100644
--- a/dist/sql-wasm.js
+++ b/dist/sql-wasm.js
@@ -173,16 +173,5 @@ if(0<K)Sa=Yc;else{if(f.preRun)for("function"==typeof f.preRun&&(f.preRun=[f.preR
   return initSqlJsPromise;
 } // The end of our initSqlJs function
 
-// This bit below is copied almost exactly from what you get when you use the MODULARIZE=1 flag with emcc
-// However, we don't want to use the emcc modularization. See shell-pre.js
-if (typeof exports === 'object' && typeof module === 'object'){
-    module.exports = initSqlJs;
-    // This will allow the module to be used in ES6 or CommonJS
-    module.exports.default = initSqlJs;
-}
-else if (typeof define === 'function' && define['amd']) {
-    define([], function() { return initSqlJs; });
-}
-else if (typeof exports === 'object'){
-    exports["Module"] = initSqlJs;
-}
+var module;
+export default initSqlJs;
